apps/mull-ui/src/app/pages/login/login.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { CustomTextInput } from '@mull/ui-lib';
+import { Link } from 'react-router-dom';
+import { History } from 'history';
+import * as Yup from 'yup';
+import { useFormik } from 'formik';
+
+import logo from '../../../assets/mull-logo.png';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFacebookSquare, faGoogle, faTwitter } from '@fortawesome/free-brands-svg-icons';
+
+import './login.scss';
+
+export interface LoginProps {
+  history: History;
+}
+
+export const Login = ({ history }: LoginProps) => {
+  const formik = useFormik({
+    initialValues: {
+      email: '',
+      password: '',
+    },
+
+    validationSchema: Yup.object({
+      email: Yup.string().required('Email is required.').email('Email format is incorrect'),
+      password: Yup.string().required('Password is required.'),
+    }),
+
+    onSubmit: (values) => {},
+  });
+
+  return (
+    <form onSubmit={formik.handleSubmit}>
+      <div className="page-container-no-nav">
+        <div className="login-container">
```

Comment:
No need to have an extra div, you should be able to apply the styling to the same div.

Reply:
oh yeah oops, was testing something forgot to merge these together, ill do this then rebase